src/test/java/unicash/logic/commands/SummaryCommandTest.java

```diff
@@ -11,13 +13,20 @@
 public class SummaryCommandTest {
     @Test
     public void execute_summary_success() {
-        Model model = new ModelManager();
-        Model expectedModel = new ModelManager();
+        Model emptyModel = new ModelManager();
+        Model expectedEmptyModel = new ModelManager();
+        CommandResult emptyModelExpectedCommandResult =
+                new CommandResult(MESSAGE_SHOWING_SUMMARY_FAILURE, false, false, false);
```

Comment:
See comment on calling this a "Failure" message, it seems more like an alternative success path